README.md

```diff
@@ -30,12 +38,52 @@ services:
       - cron.another_task.schedule="*/2 * * * *"
       - cron.another_task.command="/usr/local/bin/another_app_script.sh"
 ```
+This example shows how to schedule multiple cron jobs using cron syntax. The Docker container will run `/usr/local/bin/app_script.sh` every minute and `/usr/local/bin/another_app_script.sh` every two minutes, with logs stored in `/var/log/cron/`.
 
 ### Environment Variables
-
 The `example_compose.yml` file uses several environment variables. Make sure to define these in a `.env` file in the root directory of your project:
+- **`COMPOSE_PROJECT_NAME`**: Specifies the name of the Docker Compose project, allowing the cron to target jobs in that specific project.
+- **`DEBUG`**: Set to `true` to enable detailed output for debugging purposes.
+- **`CRON_LOG_DIR`**: Defines the directory where cron stores log files for executed jobs, defaulting to `/var/log/cron`.
+
+### Volume Mounts
+- **`/var/run/docker.sock`**: This is used to enable the Docker client inside the container to communicate with the Docker daemon running on the host. Be careful when using this as it provides elevated privileges.
+- **`./logs/cron:/var/log/cron`**: Mount a directory to store the cron logs. Make sure this folder has the appropriate permissions.
+
+## Supercronic Integration
+
+This image uses [Supercronic](https://github.com/aptible/supercronic) to run cron jobs in a more Docker-friendly way. Supercronic provides better logging, fewer overheads, and can be easily integrated with Docker containers, making it a better alternative to the traditional `cron` utility.
+
+## Managing Cron Jobs
+
+The image comes with several scripts to manage cron jobs:
+
+- **`functions.sh`**: Contains helper functions for script operations.
+- **`startup.sh`**: This script runs at container startup to initialize all required settings and start cron jobs.
+- **`update_cron.sh`**: Used for updating cron jobs dynamically without restarting the container.
+
+Make sure these scripts are executable and are properly configured according to your needs.
+
+## Building and Running the Image
```

Comment:
We should probably mention here that there exists the docker images built automatically. In the example, we use the latest version of one - `ghcr.io/wikiteq/cron`. 
All built images: https://github.com/WikiTeq/docker-cron/pkgs/container/cron